src/seed_clusterer.cpp

```diff
@@ -522,88 +538,100 @@ cerr << endl;
                 new_chain_cluster.second.insert(j);
                 for (size_t i = 0; i < new_chain_cluster_indices.size(); i++ ) {
```

Comment:
Keep clusters that extend farthest left and right at the top of the vector (smallest dist_left and dist_right)